File: javascript/moderation_text_aksk_demo.js
/**
 *  图像内容检测服务ak,sk方式请求的使用示例
 */
var text = require("./moderation_sdk/moderation_text");
var utils = require("./moderation_sdk/utils");

// 初始化服务的区域信息，目前支持华北-北京(cn-north-4)、亚太-香港(ap-southeast-1)等区域信息
utils.initRegion("cn-north-4");

var app_key = "*************";
var app_secret = "************";

text.moderation_text_aksk(app_key, app_secret, [{
    "text": "666666luo聊请+110亚砷酸钾六位qq，fuck666666666666666",
    "type": "content"
}], ["ad", "abuse", "politics", "porn", "contraband"], function (result) {
    console.log(result);
});

File: python/moderation_text_aksk_demo.py
# -*- coding:utf-8 -*-
from moderation_sdk.moderation_text import moderation_text_aksk
from moderation_sdk.utils import init_global_env

if __name__ == '__main__':
    # Services currently support North China-Beijing(cn-north-4), Asia Pacific-Hong Kong(ap-southeast-1)
    init_global_env('cn-north-4')

    #
    # access moderation text enhance,posy data by ak,sk
    #
    app_key = '*************'
    app_secret = '************'

    # call interface use the text
    result = moderation_text_aksk(app_key, app_secret, '666666luo聊请+110亚砷酸钾六位qq，fuck666666666666666', 'content')
    print(result)